ClientApp/app/(settings)/_layout.tsx

```diff
@@ -0,0 +1,12 @@
+import { Stack } from 'expo-router';
+
+export default function SettingsLayout() {
+  return (
+    <Stack screenOptions={{ headerShown: true, title: "Settings" }}>
+      <Stack.Screen name="settingsPage/index" />
```

Comment:
I don't think we need index pages for helpPage/LanguagePage/notificationsPage. Since they won't have any sub pages inside them they could just be standalone files/routes.
the only index here would be in (settings).